src/main/java/kg/jarkyn/server/Server.java

```diff
@@ -0,0 +1,33 @@
+package kg.jarkyn.server;
+
+import kg.jarkyn.server.Incoming.Requester;
+import kg.jarkyn.server.Outgoing.Responder;
+import kg.jarkyn.server.Outgoing.Response;
+
+public class Server {
+    private final ReceivingSocket serverSocket;
+    private final Responder responder;
+
+    public Server(ReceivingSocket serverSocket, Responder responder) {
+        this.serverSocket = serverSocket;
+        this.responder = responder;
+    }
+
+    public void run() {
```

Comment:
I like the high level abstractness that this class has. i.e. Listens for requests, builds a response, returns response. V simple/easy to read